### PR TITLE
fix: Add Conferencing app opens an white blank screen (CALCOM-11196)

### LINKED ISSUE #11196: Add Conferencing app opens an white blank screen
When I Click on Add Conferencing app It Shows mw blank page

### Steps to Reproduce

 Logged in and Clicked on My profile > Conferencing > click on the button "+ Add Conferencing app"

https://github.com/calcom/cal.com/assets/34880491/636c4544-d90f-4059-a62f-fd149950715f

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## DEMO

https://www.loom.com/share/ba118884b7734a76af7b48252c6d8be3

### CODE REVIEW

File: apps/web/pages/settings/my-account/conferencing.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const AddConferencingButton = () => {
 
   return (
     <>
-      <Button color="secondary" StartIcon={Plus} href="/apps/categories/video">
+      <Button color="secondary" StartIcon={Plus} href="/apps/categories/conferencing">
```
Comment (hariombalhara) [inline apps/web/pages/settings/my-account/conferencing.tsx:49]: @gitstart Can you confirm why it's not working on cal.com but working on app.cal.com as mentioned [here](https://github.com/calcom/cal.com/issues/11196#issuecomment-1710256296)

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Thanks @gitstart 🙏
Left a comment seeking clarity.

File: apps/web/pages/settings/my-account/conferencing.tsx (hunk 1 above)
Comment (Udit-takkar) [reply apps/web/pages/settings/my-account/conferencing.tsx:49]: @gitstart-calcom i don't see any problem on app.cal.com 
Screenshot 2023-09-22 at 5 54 57 PM

**Review by Udit-takkar — COMMENTED** [review]
We already have a redirect to /conferencing in apps/web/next.config.js

Screenshot 2023-09-22 at 5 55 49 PM

File: apps/web/pages/settings/my-account/conferencing.tsx (hunk 1 above)
Comment (gitstart-calcom) [reply apps/web/pages/settings/my-account/conferencing.tsx:49]: Hi @Udit-takkar 
Yeah, there's no issue on app.cal.com but on cal.com
https://www.loom.com/share/596dd405fa2e4b17a1c0d90502ecc71d

On cal.com too, works fine after a reload when it redirects to `/conferencing`

Comment (gitstart-calcom) [reply apps/web/pages/settings/my-account/conferencing.tsx:49]: We were able to reproduce the same issue locally and pushed this PR as a fix.
Please check out this demo
https://www.loom.com/share/3dbe7344a4e34ac296e5549da03e1b15

**Comment by gitstart-calcom** [comment]
> We already have a redirect to /conferencing in apps/web/next.config.js
> 
> Screenshot 2023-09-22 at 5 55 49 PM

Not sure why but this does not seem to be working @Udit-takkar 
https://github.com/calcom/cal.com/pull/11315#discussion_r1335662176

**Comment by gitstart-calcom** [comment]
Hi @Udit-takkar, this issue has been resolved through [this](https://github.com/calcom/cal.com/commit/538e3bf07c5a131afbfa6086180c8653ad2482b4#diff-8f553d36ddafd0c8c2f613fad6f4384e8c185a01566ffbd95288b219a6b34794) commit with the same change

**Comment by hariombalhara** [comment]
There is no change in the PR now 🤔
Screenshot 2023-10-06 at 13 02 05

**Pull request closed without merging**